aws/client.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
Can you move the files into `src/jinacld_tools` The it will be packaged correctly and we can install it with git.